docs/developing-locally-docker.rst

```diff
@@ -255,32 +298,39 @@ local.yml
         - django
     
     ...
+#. On the django service, confirm that you are connected to the ``envs`` files and remove/comment out the ``ports``. ::
+
+    ...
+    env_file:
+      - ./.envs/.local/.django
+    ...
+    # ports:
+    #  - 8000:8000
+    ...
 
 #. Link the ``nginx-proxy`` to ``django`` through environmental variables.
    
-   ``django`` already has an ``.env`` file connected to it. Add the following variables. You should do this especially if you are working with a team and you want to keep your local environment details to yourself.
-
-   ::
+   ``django`` already has an ``.envs/.local`` file connected to it. Add the following variables. You should do this especially if you are working with a team and you want to keep your local environment details to yourself. ::
 
       # HTTPS
-      # ------------------------------------------------------------------------------
-      VIRTUAL_HOST=my-dev-env.local
+      # ----------------------
+      VIRTUAL_HOST=my-dev-env
       VIRTUAL_PORT=8000
 
-   The services run behind the reverse proxy.
+   The django application and other services will now run behind the reverse proxy.
 
 config/settings/local.py
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-You should allow the new hostname. ::
+You should *allow* the new hostname . ::
 
-  ALLOWED_HOSTS = ["localhost", "0.0.0.0", "127.0.0.1", "my-dev-env.local"]
+  ALLOWED_HOSTS = ["localhost", "0.0.0.0", "127.0.0.1", "my-dev-env"]
 
 Rebuild your ``docker`` application. ::
 
   $ docker-compose -f local.yml up -d --build
 
-Go to your browser and type in your URL bar ``https://my-dev-env.local``
+Go to your browser and type in your URL bar ``https://my-dev-env:8800``.
```

Comment:
How does the 8800 port apply to the Docker container approach?

I think this is a typo and meant for the Django SSL Server approach.